### PR TITLE
add content for CSS display property

### PR DESCRIPTION
**Pull request by Alexandra-Haynes**
## Description

This pull request addresses the following changes and improvements:

- Refactored the CSS styles for the display property section in the Codecademy CSS documentation.
- Updated the explanations and code snippets for each `display` value.
- Added examples of use cases and common pitfalls to help learners understand the concepts better.
- Included an image illustrating the layout differences between block, inline, and inline-block elements.

## Motivation

The motivation behind these changes is to provide clearer and more comprehensive information to learners who want to understand and effectively use the CSS `display` property. By improving the documentation, we aim to enhance the learning experience and ensure that users have a solid foundation in using this fundamental CSS property.

## Impact

These changes improve the overall readability and usefulness of the CSS documentation. Learners will have a better grasp of the different `display` values and their applications. The added examples and explanations will help them make informed decisions when designing layouts and manipulating the display of elements.

I welcome any feedback and suggestions for further improvements. Thank you for reviewing this pull request!

### Type of Change

- Updating the documentation 

### Checklist

- [x ] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:187]:
```suggestion
```
I would remove everything below this line.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:114]:
```suggestion
```
You need to remove instances where you have multiple line breaks in a row. There should never be more than one line break in a row.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:116]:
```suggestion
CSS specificity allows you to override default display values of elements. Here's an example of overriding display:

```
There needs to be a line break before and after code blocks.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:130]:
```suggestion
```
Remove extra line break.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:169]:
```suggestion
```
remove this

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:153]: This could use an illustrative image.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:22]:
```suggestion
```
Remove this line

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:23]:
```suggestion
```
remove line break

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:24]:
```suggestion
```
remove line break

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:25]:
```suggestion
```
remove line break

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:20]:
```suggestion
Understanding the different values of the `display` property is crucial for building well-structured and responsive web layouts. By choosing the appropriate value, the flow of elements can be controlled, grids can be created, and the visibility of specific elements can be adjusted.
```
We do not use first or second person pronouns in Docs.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:16]:
```suggestion
```
We do not use h1 headers in Docs, nor do we have a title header. The Title field in the metadata takes care of that.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:17]:
```suggestion
```
Remove extra line break.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Alexandra-Haynes I did a first review and left a couple of comments. I suggested ways to improve the body of the article. In addition, it would be better if you take the original text from the graphic and consider how to incorporate it.

**Comment by SSwiniarski** [comment]
@Alexandra-Haynes It's been three weeks and I don't see that you've addressed a majority of my comments. Are you still working on this?

**Comment by SSwiniarski** [comment]
@KTom101, @yangc95, @caupolicandiaz any idea about @Alexandra-Haynes's problem?

**Comment by KTom101** [comment]
@Alexandra-Haynes Can you take a screenshot showing us the whole error you get?

**Comment by Alexandra-Haynes** [comment]
Screenshot of the error for $git push origin codebytes & $git push upstream codebytes

Screenshot 2023-06-07 132613

**Comment by SSwiniarski** [comment]
@KTom101, does the screenshot help?

**Comment by KTom101** [comment]
@ Alexandra-Haynes It looks like you're trying to push to the original repo instead of your forked repo. I also see `origin` and `upstream` is pointing to the same location.

What do you see when you run:

`git remote -v` 

The origin should be set to your forked repo `https://github.com/Alexandra-Haynes/docs.git`

Upstream should be set to `https://github.com/Codecademy/docs.git`

**Comment by Alexandra-Haynes** [comment]
You were correct! My origin and my upstream were pointing to the same location. I fixed and made another commit. Thank you!

**Comment by KTom101** [comment]
@Alexandra-Haynes Glad that helped! Once @SSwiniarski approves this for a second review, I will take a look next.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:19]: You need to remove the following four blank lines.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:55]:
```suggestion
## Differences between display values

```
Blank line after header

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:60]:
```suggestion
Elements with `display: block;` are rendered as block-level elements. They create a line break after the element and occupy the full width of their parent container.

```
Blank line between sentence and code block.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:67]:
```suggestion
### 2. Display: inline

```
Blank line after header

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:68]:
```suggestion
Elements with `display: inline;` are rendered as inline elements. They don't create line breaks and occupy only the necessary width and height based on their content.

```
Blank line between sentence and code block.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:77]:
```suggestion
Elements with `display: inline-block;` are rendered as inline-level block elements. They behave like inline elements but allow you to set width, height, padding, and margins.

```
blank line between sentence and code block

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:83]:
```suggestion

### 4. Display: none
```
Blank line between header and code block.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:85]:
```suggestion
Elements with `display: none;` are not rendered and are completely hidden from the page. They don't take up any space in the document flow.

```
Blank line between sentence and code block.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:91]:
```suggestion

### 5. Display: flex
```
Blank line between code block and header

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:93]:
```suggestion
Elements with `display: flex;` create a flex container, allowing you to build flexible and responsive layouts. Flex items inside the container can be positioned and resized using flexbox properties.

```
Blank line between sentence and code block.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:99]:
```suggestion

### 6. Display: grid
```
Blank line between code block and header.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:101]:
```suggestion
Elements with `display: grid;` create a grid container that enables you to define rows and columns, creating a two-dimensional grid system. Child elements inside the container can be placed in specific cells and adjusted for size and alignment.
```
Blank line between sentence and code block.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:109]: Remove 22 blank lines. (Only one blank line between the code block and the following header.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:132]:
```suggestion
## Overriding display values
```
No ending punctuation on headers

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:149]:
```suggestion
## Responsive design considerations

```
Blank line after header.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:168]:
```suggestion
## `display: grid` and `display: flex` Comparison

```

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:185]: Remove two blank lines

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:204]: Remove two blank lines.

File: content/css/concepts/display/display.md
Comment (SSwiniarski) [inline content/css/concepts/display/display.md:171]: Can you provide an image illustrating this?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Alexandra-Haynes I did another pass and had some comments, a lot having to do with proper spacing of elements. Basically there needs to be one, and only one, blank line between paragraphs, code blocks, and headers.

File: content/css/concepts/display/display.md
Comment (Alexandra-Haynes) [reply content/css/concepts/display/display.md:183]: Yes, I want to add this image, together with a proper description and the code.

Illustrating display flex versus display grid


I checked the documentation to make sure I am using the correct format and still my image is not being displayed. The image is located in the media folder and I inserted it using 

_Display Flex Versus Display Grid Example_


Still is not being displayed. Can you please help me understand what am I doing wrong? Thank you!

Comment (KTom101) [reply content/css/concepts/display/display.md]: @Alexandra-Haynes You've linked it in properly. It won't display right now because it's not merged. Once merged, the image link will work and it will be displayed on the page.

Comment (Alexandra-Haynes) [reply content/css/concepts/display/display.md]: That makes sense! Thank you for the fast response, I appreciate your help

**Review by SSwiniarski — APPROVED** [review]
@Alexandra-Haynes Looks good for a second review.

File: content/css/concepts/display/display.md
Comment (KTom101) [inline content/css/concepts/display/display.md:156]: For the tabbing in this example use 2 spaces instead of using tabs. So all the lines in the example will shift to the left after this is done.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Alexandra-Haynes I've added some changes.

File: content/css/concepts/display/display.md
Comment (KTom101) [inline content/css/concepts/display/display.md:199]:
```suggestion
## Example 
```

File: content/css/concepts/display/display.md
Comment (KTom101) [inline content/css/concepts/display/display.md:85]:
```suggestion
### 4. `display: none`
```

File: content/css/concepts/display/display.md
Comment (KTom101) [inline content/css/concepts/display/display.md:95]:
```suggestion
### 5. `display: flex`
```

**Review by KTom101 — COMMENTED** [review]
@Alexandra-Haynes I see you've made some changes. I've added a couple more. Don't forget the change on line 157!

**Review by KTom101 — APPROVED** [review]
@Alexandra-Haynes LGTM 👍

**Comment by KTom101** [comment]
@Alexandra-Haynes I approved the tests to run at the bottom. I noticed the formatting check failed so I ran the `npm run format:all` command to correct it. If needed, you can read more about the CI workflow [here](https://www.codecademy.com/resources/docs/contribution-guide/getting-set-up-on-your-computer#:~:text=a%20new%20file.-,Additional%20set%20up%20needed,-There%20are%20continuous) for your next entry. Thanks!

**Comment by KTom101** [comment]
@Alexandra-Haynes The changes are live: https://www.codecademy.com/resources/docs/css/display


You're currently listed as anonymous. 

Screenshot 2023-06-26 at 3 19 48 PM

If you want to remain anonymous, there is nothing more you need to do. If you wish to be listed as a contributor, please follow the instructions linked [here](https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036).

cc @yangc95

**Pull request merged**
